---
std_msgs/Bool register_map